recipes/mcp-server-fetch/recipe.yaml

```
@@ -0,0 +1,55 @@
+context:
+  name: mcp-server-fetch
+  version: 2025.1.17
+  python_min: 3.10.0
```

```suggestion
  python_min: 3.10
```
Keep this looser so that we can test using newer minor releases.